### PR TITLE
PWX-23476 update rancher catalog to 2.9.1.4

### PR DESCRIPTION
**Pull request by dahuang-purestorage**
Signed-off-by: dahuang <dahuang@purestorage.com>

**What this PR does / why we need it**:
Update rancher charts to 2.9.1.4
**Which issue(s) this PR fixes** (optional)
PWX-23476

**Special notes for your reviewer**:
tested deployment locally

### CODE REVIEW

File: charts/portworx/v2.9.1/values.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ serviceAccount:
 
 deploymentType: oci                   # accepts "oci" or "docker"
 imageType: none                       #
-imageVersion: 2.8.0                   # Version of the PX Image.
+imageVersion: 2.9.1.4                   # Version of the PX Image.
```
Comment (trierra) [inline charts/portworx/v2.9.1/values.yaml:40]: apparently we're one version behind. We need to add the 2.10.0 too

File: stable/index.yaml [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+    - email: dahuang@purestorage.com
+      name: dahuang-purestorage
+    name: portworx
+    sources:
+    - https://github.com/portworx/charts-rancher/tree/master/stable
+    urls:
+    - https://raw.githubusercontent.com/portworx/charts-rancher/master/stable/portworx-2.9.1.tgz
+    version: 2.9.1
   - apiVersion: v1
     appVersion: "2.8"
```
Comment (trierra) [inline stable/index.yaml:36]: why several appVersion with different appVersion values?

File: stable/index.yaml [hunk 3]
```diff
@@ -1,9 +1,40 @@
 apiVersion: v1
 entries:
   portworx:
+  - apiVersion: v1
+    appVersion: 2.9.1
```
Comment (trierra) [inline stable/index.yaml:5]: let's add the 2.10.0 after this. We should match the latest central version 
You can use [this](https://install.portworx.com/2.10?mc=false&kbver=1.20.0&b=true&kd=type%3DPremium_LRS%2Csize%3D150&s=%22type%3DPremium_LRS%2Csize%3D50%22&c=px-cluster-19d04198-1185-4fce-aea7-adf3adef6f05&aks=true&stork=true&csi=true&mon=true&tel=false&st=k8s) spec to check the latest versions

**Review by trierra — CHANGES_REQUESTED** [review]
the stork versions in questions and values don't match. The latest stork version is         `image: openstorage/stork:2.9.0`

**Comment by dahuang-purestorage** [comment]
similar to helm. i think we just need to update to 2.9.1.4

**Review by trierra — APPROVED** [review]
yes, no need to upgrade to 2.10.0. Let's get this it

**Pull request merged**